Client registration #9333 ke (#36)
uyuni-project/uyuni-docs: modules/client-configuration/pages/registration-overview.adoc

[[registration-overview]]
= SLE Client Registration Overview

There are several ways to register clients to your {productname} Server.
The preferred methods are described in the following sections:

> * For Salt clients, it is recommended to register your clients using the {productname} {webui}.

Reviewer: ```suggestion
* For Salt clients, we recommend that you register clients using the {productname} {webui}.
```

For more information, see xref:client-configuration:registration-webui.adoc[].

> * If you want more control over the process, have to register many clients, or are registering traditional clients, it is recommended to create a bootstrap script.

Reviewer: ```suggestion
* If you want more control over the process, have to register many clients, or are registering traditional clients, we recommend that you create a bootstrap script.
```
